src/dynimage.rs

```diff
@@ -940,7 +941,7 @@ impl DynamicImage {
     ///
     /// ```
     /// # fn only_check_if_this_compiles() -> Result<(), Box<dyn std::error::Error>> {
-    /// use image::{Orientation, DynamicImage, ImageReader, ImageDecoder};
+    /// use image::{DynamicImage, ImageReader, ImageDecoder, metadata::Orientation};
```

Comment:
I think we can actually just remove the import of `Orientation` here, since the example just calls `decoder.orientation()` to construct one

Reply:
Good call, it's a shame the compiler doesn't surface warnings from doctests